types/crypto_test.go

```diff
@@ -6,6 +6,10 @@ import (
 	"testing"
 )
 
+func init() {
+	bls.Init(0)
```

Comment:
Does `0` mean that we need to use `MCL_BN254` curve type? Why not `bls.BLS12_381`?

Reply:
Was removed, should be handled outside of this PR scope